_build/src/dep/GUM_news_lanterns.conll10

```diff
@@ -145,34 +145,34 @@
 15	luck	_	NN	NN	_	13	dobj	_	_
 16	.	_	SENT	.	_	0	punct	_	_
 
-1	Started	_	VVN	VBN	_	9	vmod	_	_
-2	in	_	IN	IN	_	1	prep	_	_
-3	1990	_	CD	CD	_	2	pobj	_	_
-4	,	_	,	,	_	0	punct	_	_
-5	the	_	DT	DT	_	8	det	_	_
-6	Taipei	_	NP	NNP	_	8	nn	_	_
-7	lantern	_	NN	NN	_	8	nn	_	_
-8	festival	_	NN	NN	_	9	nsubj	_	_
-9	continues	_	VVZ	VBZ	_	0	root	_	_
-10	a	_	DT	DT	_	14	det	_	_
-11	2000	_	CD	CD	_	12	num	_	_
-12	year	_	NN	NN	_	13	npadvmod	_	_
-13	old	_	JJ	JJ	_	14	amod	_	_
-14	tradition	_	NN	NN	_	9	dobj	_	_
-15	of	_	IN	IN	_	14	prep	_	_
-16	light	_	NN	NN	_	17	mwe	_	_
-17	related	_	JJ	JJ	_	18	amod	_	_
-18	festivals	_	NNS	NNS	_	15	pobj	_	_
-19	marking	_	VVG	VBG	_	18	vmod	_	_
-20	the	_	DT	DT	_	21	det	_	_
-21	end	_	NN	NN	_	19	dobj	_	_
-22	of	_	IN	IN	_	21	prep	_	_
-23	the	_	DT	DT	_	27	det	_	_
-24	Chinese	_	NP	NNP	_	26	amod	_	_
-25	New	_	NP	NNP	_	26	nn	_	_
-26	Year	_	NP	NNP	_	27	nn	_	_
-27	celebrations	_	NNS	NNS	_	22	pobj	_	_
-28	.	_	SENT	.	_	0	punct	_	_
+1	Started	_	VVN	_	_	9	vmod	_	_
+2	in	_	IN	_	_	1	prep	_	_
+3	1990	_	CD	_	_	2	pobj	_	_
+4	,	_	,	_	_	0	punct	_	_
+5	the	_	DT	_	_	8	det	_	_
+6	Taipei	_	NP	_	_	8	nn	_	_
+7	lantern	_	NN	_	_	8	nn	_	_
+8	festival	_	NN	_	_	9	nsubj	_	_
+9	continues	_	VVZ	_	_	0	root	_	_
+10	a	_	DT	_	_	14	det	_	_
+11	2000	_	CD	_	_	12	num	_	_
+12	year	_	NN	_	_	13	npadvmod	_	_
+13	old	_	JJ	_	_	14	amod	_	_
+14	tradition	_	NN	_	_	9	dobj	_	_
+15	of	_	IN	_	_	14	prep	_	_
+16	light	_	NN	_	_	18	amod	_	_
```

Comment:
the amod is "related", light is its modifier (I'd take 'nn' as the function, the Stanford converter does the same structure with 'dep')